test/input_with_schedules/closeness_centrality_unweighted.gt

```diff
@@ -0,0 +1,5 @@
+
+
```

Comment:
rename this file with a more meaningful name indicating the schedule. For example, closeness_centrality_SparsePushDensePull_parallel.